ChangeLog

```diff
@@ -31,6 +31,18 @@ API Changes
      mbedtls_ctr_drbg_update() -> mbedtls_ctr_drbg_update_ret()
      mbedtls_hmac_drbg_update() -> mbedtls_hmac_drbg_update_ret()
    * Extend ECDH interface to enable alternative implementations.
+   * Deprecate the ARIA error constants
```

Comment:
Minor: I would consider rephrasing as 
>Deprecate the ARIA error constants MBEDTLS_ERR_ARIA_INVALID_KEY_LENGTH and MBEDTLS_ERR_ARIA_INVALID_INPUT_LENGTH in favour of...

To keep the ChangeLog look and feel

Reply:
Do we have a strict character limit for the ChangeLog?

Reply:
I am not sure. As I said, this is minor, and I wouldn't block on this.
This usage of `-` is different than all other entries, and this is why I added the comment.

Reply:
> Do we have a strict character limit for the ChangeLog?
We do wrap at 79 cols, if that's what you mean, but if you don't, I doubt we do.